apps/examples/src/examples/keyboard-shortcuts/KeyboardShortcuts.tsx

```diff
@@ -9,6 +9,7 @@ const overrides: TLUiOverrides = {
 	//[a]
 	actions(_editor, actions): TLUiActionsContextType {
 		actions['toggle-grid'].kbd = 'x'
+		actions['copy-as-png'].kbd = '$1'
```

Comment:
This is going to cause problems for other examples / e2e tests. Rather than mutating the actions, we should be returning a new actions. So something like:

```
const newActions = [...actions]
newActions['toggle-grid'] = {...newActions['toggle-grid'], kbd: "x"}
return newActions
```